Tribute_Page/style.css

```diff
@@ -26,11 +26,11 @@ header p {
     font-size: 1.6rem;
     text-align: center;
 }
-.container {
+#container {
     background-color:white;
     padding: 1px;
 }
-.container p {
+#container p {
     text-align: center;
     font-size: 1.8rem;
```

Comment:
your CSS selectr option is very good, keep it up, and also please apply the linter error assurance, you are not applied

Reply:
> your CSS selectr option is very good, keep it up, and also please apply the linter error assurance, you are not applied


You must have meant '...you have not applied it.'😊

- [ ] Let us always try to suggest changes politely. "I think it would be better to add.." would have been a better approach. Also, it is always important to add the WHY, this is for the student to know the reason you are requesting changes.

- [ ]